src/NHibernate/Mapping/Table.cs

```diff
@@ -194,6 +190,7 @@ public string Schema
 		public int UniqueInteger
 		{
 			get { return uniqueInteger; }
+			internal set { uniqueInteger = value; }
```

Comment:
Since this member now depends on being externally supplied (be it by setter or constructor), maybe should we guard against it being used without having been set.
I would do it by switching the backing field to nullable then adjusting the getter to something like `get => uniqueInteger ?? throw new InvalidOperation("UniqueInteger has not been supplied");`. (If using expression bodied getter, change also the setter for consistency.)

Reply:
@fredericDelaporte Good idea. Will do it. And before making any changes in ctor... Are you sure about it? ) I see no troubles with this change but I just don't see why to fight to keep `uniqueInteger` readonly. Considering that it's the only read-only field in this class.

Reply:
Forget about the constructor. I mentioned it just to tell that if this member value was supplied through a constructor, I would still consider doing this check in the getter.